app/routes/index.js

```diff
@@ -0,0 +1,22 @@
+var express = require('express');
+var router = express.Router();
+var formBuilder = require('../tools/formTool.js');
+
+var input1 = {field: 'input', type: 'text', id: 'firstName', label: 'First Name'};
+var input2 = {field: 'input', type: 'text', id: 'lastName', label: 'Last Name'};
+var select = {field: 'select', id: 'genderSelect', label: 'Gender', options: ['male', 'female'] };
+var button = {field: 'button', type: 'submit', id: 'submitBtn', label: 'Submit'};
+
+
+/* GET home page. */
+router.get('/', function(req, res, next) {
+  var form = formBuilder(input1, input2, select, button);
```

Comment:
You can then change the formBuilder method to accept the entire JSON array instead of hard coding the inputs into the method signature.